R/compiled.R

```diff
@@ -22,20 +22,45 @@ parse_gcov <- function(file, package_path = "") {
     ":"
   )
 
+  re_b <- rex::rex("branch  ",
```

Comment:
Can't you have a single regex with a capture group that is either digits or "never execute". Then based on the result, you know if it was taken or not

Reply:
I tried, but none worked. I tried with ||, anyof, and someof (which are what I can remember now). You are welcome to try!

Reply:
I was able to combine them using %or% and or().